derive_builder_test/tests/try_from.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "nightlytests")]
+#![cfg_attr(feature = "nightlytests", feature(try_from))]
```

Comment:
please merge (or rebase to) current master. Then depend on `feature = "try_from"` instead.